java/samples/sample-code/src/main/java/com/microsoft/semantickernel/samples/syntaxexamples/Example06_TemplateLanguage.java

```diff
@@ -105,6 +105,6 @@ Is it weekend time (weekend/not weekend)?
         // Show the result
         System.out.println("--- Prompt Function result");
         var result = kernel.invokeAsync(kindOfDay, null, String.class).block();
-        System.out.println(result.getResultVariable());
+        System.out.println(result.getResultVariable().getValue());
```

Comment:
I thought there was a "getValue()" on FunctionResult that is a shorthand for this.
`<edit>`I see you do this in Example09. Best to be consistent. It doesn't matter to me which way.`</edit>`